conans/test/cmake_test.py

```diff
@@ -1,10 +1,18 @@
+import shutil
+import sys
+import tempfile
 import unittest
 from conans.model.settings import Settings
 from conans.client.conf import default_settings_yml
 from conans.client.cmake import CMake
 
 
 class CMakeTest(unittest.TestCase):
+    def setUp(self):
+        self.tempdir = tempfile.mkdtemp()
```

Comment:
`CMake.configure` will actually call `mkdir()`, that's why I need a valid directory name and then remove it.